### PR TITLE
test-clang.sh: use unsecure seccomp to work around runc issue

### LINKED ISSUE #34: Failed to download metadata for repo 'fedora': Cannot prepare internal mirrorlist: Curl error (6): Couldn't resolve host name for
```
$ make epoch3
...
[+] Testing clang in 'docker.io/fedora:latest' container
[+] Testing 'clang --version'
clang version 14.0.1
Target: x86_64-alpine-linux-musl
Thread model: posix
InstalledDir: /repo/toolchain/bin
[+] Updating OS
Error: Failed to download metadata for repo 'fedora': Cannot prepare internal mirrorlist: Curl error (6): Couldn't resolve host name for https://mirrors.fedoraproject.org/metalink?repo=fedora-36&arch=x86_64 [getaddrinfo() thread failed to start]
make: *** [Makefile:32: epoch3] Error 1
```
cc @nathanchance

### PR DESCRIPTION
**Pull request by nickdesaulniers**
Otherwise we observe the error:

[+] Updating OS
Error: Failed to download metadata for repo 'fedora': Cannot prepare
internal mirrorlist: Curl error (6): Couldn't resolve host name for
https://mirrors.fedoraproject.org/metalink?repo=fedora-36&arch=x86_64
[getaddrinfo() thread failed to start]

Link: https://pascalroeleven.nl/2021/09/09/ubuntu-21-10-and-fedora-35-in-docker/
Link: https://bugzilla.redhat.com/show_bug.cgi?id=2009047
Fixes #34

### CODE REVIEW

**Review by nathanchance — APPROVED** [review]
:(

**Comment by nickdesaulniers** [comment]
https://github.com/ClangBuiltLinux/containers/issues/34#issuecomment-1152800492

**Comment by nickdesaulniers** [comment]
added another commit on top, PTAL @nathanchance

File: llvm-project/Makefile [hunk 1]
```diff
@@ -35,3 +34,6 @@ clang: epoch3
 	$(DOCKER) rm llvm-project || true
 	$(DOCKER) create --name llvm-project $(IMAGE):stage3-$(HOST_ARCH)
 	$(DOCKER) cp llvm-project:/usr/local/bin/clang-14 clang
+
+test: clang
```
Comment (nathanchance) [inline llvm-project/Makefile:38]: I think `epoch3` would be a better prerequisite, as we are testing the container, not the binary?

**Review by nathanchance — APPROVED** [review]

**Pull request merged**